R/Figures.R

```diff
@@ -1380,23 +1376,23 @@ TADA_GroupedScatterplot <- function(.data, group_col = "MonitoringLocationName",
         plot_bgcolor = "#e5ecf6",
         margin = mrg,
         legend = list(
+          title = list(text = paste0('<b>', group_col,'<b>'), x = 0.5, y= 100),
```

Comment:
@hillarymarler over here for legend title

Reply:
Thanks!